src/main/java/com/coherent/training/selenium/stanila/pages/LoginPOM.java

```diff
@@ -1,49 +1,57 @@
 package com.coherent.training.selenium.stanila.pages;
-
-import com.coherent.training.selenium.stanila.Waits;
-import com.coherent.training.selenium.stanila.base.BaseClass;
+import com.coherent.training.selenium.stanila.base.BasePOM;
+import com.coherent.training.selenium.stanila.waits.Waits;
 import lombok.SneakyThrows;
+import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
-import org.openqa.selenium.support.PageFactory;
 
-public class LoginPOM extends BaseClass{
+public class LoginPOM extends BasePOM {
     @FindBy (xpath = "//button[starts-with(@class,'Button2')]")
-    private WebElement LOGIN_CENTRAL_PAGE;
+    private WebElement loginFirst;
     @FindBy (xpath ="//input[@id='passp-field-login']")
-    private WebElement USERNAME_INPUT;
+    private WebElement usernameInput;
     @FindBy (xpath = "//button[@id='passp:sign-in']")
```

Comment:
loginUsername and loginPassword have the same locator, why don't you use just one web element?